### PR TITLE
Add detection support of TCL versions 8.6 and 8.5

### PR DESCRIPTION
**Pull request by dpino**
In addition to adding labels "8.6" and "8.5" to the list of possible TCL versions, the patch also changes how `libtcl.so`if found. In my system (Ubuntu 18.04) `libtcl8.6.so` lives at `/usr/lib/x86_64-linux-gnu/`. Trying to figure out why is that I learned Debian and Ubuntu are moving now to a new multiarch implementation where mixed-architecture installations can be managed much more sanely [1], thus this new directory layout.

Basically the change does a recursive file search over two root directories and removes the dirname, leaving only the basename of the path. The rest is the same as before.

Regarding version 8.5, I added the label but I couldn't test it (since my system features 8.6).

[1] https://unix.stackexchange.com/questions/43190/where-did-usr-lib64-go-and-what-is-usr-lib-x86-64-linux-gnu

### CODE REVIEW

**Comment by dpino** [comment]
I just noticed https://github.com/antirez/hping/pull/13. I tested it and it doesn't work in my system, because of the new directory layout. I'm fine closing this PR, but #13 would need to be updated.
